autoware_launch/launch/components/tier4_autoware_api_component.launch.xml

```diff
@@ -3,5 +3,23 @@
   <!-- Fork the repository and add the parameters here if needed. -->
   <arg name="scenario_simulation" default="false"/>
   <arg name="launch_deprecated_api" default="$(var scenario_simulation)"/>
+  <arg name="use_sim_time" default="false"/>
+  <arg name="vehicle_model" default="sample_vehicle"/>
+  <arg
+    name="component_wise_launch"
+    default="false"
+    description="Launch in a component-wise manner.
+    If true, this launch file will be launched in a component-wise manner.
+    If false, this launch file will be launched in the autoware.launch.xml."
+  />
+
+  <!-- Global parameters -->
+  <group if="$(var component_wise_launch)" scoped="false">
```

Comment:
Do you know what happens if we launch global_params.launch.py in both component launch file and top level autoware.launch.xml? Does it produce some kind of an error?

If the behavior won't change, maybe we could remove the component_wise_launch argument.

Reply:
https://github.com/autowarefoundation/autoware_launch/pull/1469/commits/51aba6a1453c2320e493858099561a4e9d3067fd